src/main/java/com/sap/oss/phosphor/fosstars/model/value/ArtifactVersion.java

```diff
@@ -27,19 +27,29 @@
  */
 public class ArtifactVersion {
 
-  public static final ArtifactVersion EMPTY = new ArtifactVersion("", LocalDate.now());
+  /**
+   * Default immutable EMPTY artifact version.
+   */
+  public static final ArtifactVersion EMPTY =
+      new ArtifactVersion("-", LocalDateTime.parse("1970-01-01T01:00:00"));
```

Comment:
Not a big deal but may be you can just this constant.

```suggestion
      new ArtifactVersion("-", LocalDateTime.MIN);
```